.github/workflows/tests.yaml

```diff
@@ -19,7 +19,7 @@ jobs:
       - name: Setup Python
         uses: actions/setup-python@v5
         with:
-          python-version: '3.11'
+          python-version: '3.10'
```

Comment:
Should we run linting against newest or oldest interpreter?

Reply:
It doesn't really matter. In fact we could probably remove the `setup-python` step from linting and just use the system python if we wanted.

Reply:
I removed `Setup Python` action from linting. Let's see how it ends